roles/aws_eks/tasks/main.yml

```diff
@@ -14,9 +15,11 @@
 
 - name: Create EKS cluster
   shell: |
-    aws eks create-cluster --name {{ aws_project_name }} --role-arn {{ eks_role }} --resources-vpc-config subnetIds={{ vpc.stack_outputs.SubnetIds }},securityGroupIds={{ vpc.stack_outputs.SecurityGroups }}
+    aws eks create-cluster --name {{ aws_project_name }} --role-arn {{ eks_role }} --resources-vpc-config subnetIds={{ vpc.stack_outputs.SubnetIds }},securityGroupIds={{ vpc.stack_outputs.SecurityGroups }} --kubernetes-version 1.14
```

Comment:
Version hardcoded, as templates could be dependent on it, not something to change lightly.